cmd/issuer-rest/startcmd/start.go

```diff
@@ -826,3 +846,19 @@ func getOauth2Config(authURL, tokenURL, redirectURL, clientID, secret string) *o
 
 	return config
 }
+
+func readProfiles(path string) ([]*operation.Profile, error) {
+	f, err := os.Open(path) //nolint:gosec
+	if err != nil {
+		return nil, fmt.Errorf("open file with profiles: %w", err)
+	}
+	defer f.Close() //nolint:gosec,errcheck
+
+	var profiles []*operation.Profile
```

Comment:
Unless you have a specific reason to, you don't generally need an array of pointers - you can change this to `[]operation.Profile`